docker-generator/src/main/java/org/ballerinax/docker/generator/DockerArtifactHandler.java

```diff
@@ -292,7 +361,35 @@ private String generateDockerfile() {
             dockerfileContent.append("USER ballerina").append("\n");
             dockerfileContent.append("\n");
         }
+    }
 
+    /**
+     * Generate Dockerfile content.
+     *
+     * @return Dockerfile content as a string
+     */
+    private String generateDockerfile() {
+        StringBuilder dockerfileContent = new StringBuilder();
+        dockerfileContent.append("# Auto Generated Dockerfile\n");
+        dockerfileContent.append("FROM ").append(this.dockerModel.getBaseImage()).append("\n");
+        dockerfileContent.append("\n");
+        dockerfileContent.append("LABEL maintainer=\"dev@ballerina.io\"").append("\n");
+        dockerfileContent.append("\n");
+
+        if (this.dockerModel.getBaseImage().equals(DockerGenConstants.OPENJDK_8_JRE_ALPINE_BASE_IMAGE)) {
+            dockerfileContent.append("RUN addgroup troupe \\").append("\n");
+            dockerfileContent.append("    && adduser -S -s /bin/bash -g 'ballerina' -G troupe -D ballerina \\")
+                    .append("\n");
+            dockerfileContent.append("    && apk add --update --no-cache bash \\").append("\n");
+            dockerfileContent.append("    && chown -R ballerina:troupe /usr/bin/java \\").append("\n");
+            dockerfileContent.append("    && rm -rf /var/cache/apk/*").append("\n");
+            dockerfileContent.append("\n");
+        }
```

Comment:
This is already there in the thin jar Dockerfile. Do we need to duplicate it ?

Reply:
This is to generate uber jar Dockerfile.